Assets/GothicVR/Scripts/Manager/GvrSceneManager.cs

```diff
@@ -26,80 +32,146 @@ public class GvrSceneManager : SingletonBehaviour<GvrSceneManager>
         /// </summary>
         public void LoadStartupScenes()
         {
-            generalScene = SceneManager.LoadScene(generalSceneName, new LoadSceneParameters(LoadSceneMode.Additive));
-            SceneManager.MoveGameObjectToScene(interactionManager, generalScene);
-
-            LoadWorld("world.zen", "FP_PICKRICE_SWAMP_02");
+            LoadWorld("oldmine.zen", "ENTRANCE_SURFACE_OLDMINE");
```

Comment:
Can you please add a FeatureFlag-property where we have a dropdown of ~5 well known spawn points? This would be better than hard coding it here.